src/components/profile/Main.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import classes from '@/components/profile/style.css'
+import ThumbUpAltIcon from '@mui/icons-material/ThumbUpAlt'
+import Button from '@mui/material/Button'
+import { useLocation } from 'react-router-dom'
+import { UserInfoContentType, UserInfoType } from '@/utils/types'
+
+type UserIdInfo = {
+  id: string
+}
+
+export const Main = () => {
+  const location = useLocation()
+  const [selectUser, setSelectUser] = React.useState(location.state as UserIdInfo)
+  const [user, setUser] = React.useState({} as UserInfoContentType)
+  const [userLikedNum, setUserLikedNum] = React.useState(0)
+  const [likeButtonDisable, setLikeButtonDisable] = React.useState(false)
+
+  const fetchSelectUser = async (): Promise<void> => {
+    const response = await fetch(`${process.env.API_ENDPOINT}/profile`, {
+      method: 'GET',
+    })
+    const resJson: UserInfoType = await response.json()
+    const user = resJson[selectUser.id]
+    setUser(user)
+    setUserLikedNum(user.likedNum)
+  }
+
+  React.useEffect(() => {
+    fetchSelectUser().catch((error) => console.error(error))
+  }, [])
+
+  const likeCountUp = async () => {
+    const putParam = {
+      likedNum: userLikedNum + 1,
+    }
+    const parameter = {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(putParam),
+    }
+    const response = await fetch(`${process.env.API_ENDPOINT}/profile?userId=${selectUser.id}`, parameter)
+    const resJson: { message: string } = await response.json()
+
+    if (resJson.message === 'OK') {
```

Comment:
あまり経験ないですが、
```suggestion
    if (resJson.status === 200) {
```
の方が一般的かな？と思いました！
messageは、アプリによって返すかどうか分かれる気がします！

Reply:
@kan-mayoshi 
修正しました